xtask/src/dataset.rs

```diff
@@ -0,0 +1,165 @@
+use crate::log::init_log;
+use anyhow::anyhow;
+use clap::Parser;
+use postgresql_commands::{pg_dump::PgDumpBuilder, CommandBuilder};
+use serde_json::Value;
+use std::{io::BufReader, path::PathBuf};
+use tokio::io::AsyncWriteExt;
+use trustify_common::db;
+use trustify_module_importer::{
+    model::{CommonImporter, CsafImporter, CveImporter, ImporterConfiguration, SbomImporter},
+    server::{context::RunContext, ImportRunner},
+};
+use trustify_module_storage::service::fs::FileSystemBackend;
+
+#[derive(Debug, Parser)]
+pub struct GenerateDump {
+    /// The name of the output dump file
+    #[arg(short, long, default_value = "dump.sql")]
+    output: PathBuf,
+
+    /// The name of the input configuration. Uses a default configuration if missing.
+    #[arg(short, long)]
+    input: Option<PathBuf>,
+}
+
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub struct Instructions {
+    import: Vec<ImporterConfiguration>,
+}
+
+impl GenerateDump {
```

Comment:
The goal was to have a process that can generate a dump that you (or the CI) can then import/load again. Instead of doing that with a manual instance, having a CLI/xtask to perform all the sequential steps automatically.

Reply:
ok makes sense, thx!